packages/mobile/src/utils/theme.ts

```diff
@@ -134,6 +142,10 @@ export const matrixTheme = {
   ...darkTheme,
   background: '#1A1818',
   backgroundSecondary: '#1A1818',
+  backgroundSurface: '#1D211B',
```

Comment:
do we have more canonical names for these anywhere?

Reply:
Do you mean should these map directly to `neutral` variants? Otherwise, `backgroundSurface` and `backgroundSurface2` are the semantic color names in our Harmony specs. (cc @dylanjeffers in case I'm wrong!)